### PR TITLE
Task 9354

### PR DESCRIPTION
**Pull request by vanpd-0859**
### Issue Link :link:

https://edu-redmine.sun-asterisk.vn/issues/9354
### Goals :soccer:

- Click Done button to go to Wallet screen

### CODE REVIEW

**Comment by vanpd-0859** [comment]
ready

File: CryptoWallet/Scenes/Login/CreateWallet/BackupNewWallet/BackupNewWalletViewController.swift [hunk 1]
```diff
@@ -10,8 +10,22 @@ import UIKit
 import Reusable
 
 final class BackupNewWalletViewController: UIViewController {
+    @IBOutlet weak var mnenomicLabel: UILabel!
```
Comment (tienpx3010) [inline CryptoWallet/Scenes/Login/CreateWallet/BackupNewWallet/BackupNewWalletViewController.swift]:
```suggestion
    @IBOutlet private weak var mnenomicLabel: UILabel!
```

Comment (vanpd-0859) [reply CryptoWallet/Scenes/Login/CreateWallet/BackupNewWallet/BackupNewWalletViewController.swift]: Đã fix rồi a :D

**Comment by vanpd-0859** [comment]
ready

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**